web/src/scripts/webflow/methods.mjs

```diff
@@ -56,38 +56,33 @@ const getAllCollections = async () => {
 const createItem = (item, collectionId, draft = true) => {
   return axios({
     method: "post",
-    url: `https://api.webflow.com/collections/${collectionId}/items`,
+    url: `https://api.webflow.com/v2/collections/${collectionId}/items`,
     headers: { ...headers, "content-type": "application/json" },
     data: {
-      fields: {
-        _archived: false,
-        _draft: draft,
-        ...item,
-      },
+      isArchived: false,
+      isDraft: draft,
```

Comment:
I updated these keys as per version 2 of the Webflow API. The documentation link is in the description, along with a complete list of changes.